tests/topotests/bgp_route_map_on_match_next/test_bgp_route_map_on_match_next.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+#
+# test_bgp_route_map_on_match_next.py
+#
+# Copyright (c) 2022 Rubicon Communications, LLC.
+#
+# Permission to use, copy, modify, and/or distribute this software
```

Comment:
Please drop all the text below, leave only Copyright. Plus, add the SPDX-License tag (see other examples).

Reply:
Done